packages/ubus-lime-utils/tests/test_hotspot_wwan.lua

```diff
@@ -71,6 +88,67 @@ describe('hotspot_wwan tests #hotspot_wwan', function()
         assert.is.equal(-66, status.signal)
     end)
 
+    it('test is_safe is false when no mesh ifaces configured', function()
```

Comment:
Shouldn't be "test is_safe is false when mesh iface configured in the same radio"?

Reply:
yes